pkg/utils/names/names.go

```diff
@@ -36,3 +38,7 @@ func GetFloatingAddressClaimName(openStackMachineName string) string {
 func GetOpenStackMachineNameFromClaimName(claimName string) string {
 	return strings.TrimSuffix(claimName, fmt.Sprintf("-%s", FloatingAddressIPClaimNameSuffix))
 }
+
+func ClusterName(cluster *clusterv1.Cluster) string {
```

Comment:
Maybe we should call this variable something else? I get this is used for a lot of resources however it is not the clusters name.

Maybe clusterResourceName or something could work?

I really like the initiative to move this to a function since the name are present in a lot of places in the code.

Reply:
I've pushed a separate commit which changes it everywhere. Note that `clusterName` in the e2e tests actually does mean cluster name, so I haven't changed it there.